tests/integration/test_schema.py

```diff
@@ -1186,7 +1186,7 @@ async def assert_schema_versions(client: Client, url_trail: str, schema_id: int,
     assert res.json() == [3]  # Version number generation should now begin at 3
 
     # Check the return format on a more complex schema for version get
-    subject_2 = new_random_name("subject")
+    s2 = new_random_name("subject")
```

Comment:
I would argue that we should prefer longer names instead of shorter lines. Also none of the lines here were long enough for this to be necessary